tests/test_info.py

```diff
@@ -151,3 +142,9 @@ def test_get_cpu_temp():
         assert get_cpu_temp() == 76.543
         m.return_value.__enter__.side_effect = FileNotFoundError
         assert get_cpu_temp() == 0.0
+
+
+def test_get_pi_info():
+    assert get_pi_info('a02082') == PiInfo('3B', '1.2', 'BCM2837', 'Sony', '1Gb')
```

Comment:
Should memory not be measured in GB? (I've made a similar comment before in gpiozero)

Reply:
Yes :)